routes/modelIndex.js

```diff
@@ -79,6 +78,41 @@ var route = function (req, res, next) {
             plural: req.linz.model.linz.formtools.model.plural
         },
         modelQuery: JSON.stringify(req.linz.model.formData)
+    };
+
+    const renderRecordAction = data.model.list.recordActions.renderer || recordActionRenderers.defaultRenderer;
+
+    data.records.forEach((record, index) => {
+
+        renderRecordAction({
```

Comment:
@smebberson Yeah that seems a bit quicker, I think it may also have been the networking in virtualbox. The other screens seem slow sometimes too.